packages/playwright-core/src/server/injected/xpathSelectorEngine.ts

```diff
@@ -21,7 +21,7 @@ export const XPathEngine: SelectorEngine = {
     if (selector.startsWith('/'))
       selector = '.' + selector;
     const result: Element[] = [];
-    const document = root instanceof Document ? root : root.ownerDocument;
+    const document = root.ownerDocument || root;
```

Comment:
Should we add `Document/Node` to the list of prohibited globals?

Reply:
I did at first, but I could not find a way to ban it only as a runtime object - it yells at all the type declarations...